### ISSUE TITLE
File copy improvements

### ISSUE THREAD

**Issue by JosephTLyons** [zed-industries/feedback#712]
### Check for existing issues

- [X] Completed

### Is your feature request related to a problem?

-

### Describe the solution you'd like

- Holding down `opt` when dragging and dropping a file copies it, rather than cuts it
- When copying a file, automatically put the user in file name edit mode
  - Maybe still pre-populate the field with text in the same way as it works now, but drop the cursor in at the end of the test
- Add duplicate action in project panel context menu

**Comment by kwando** [comment]
Yes! A "duplicate file" would be nice in the context  / "right click menu" in the project panel. Would also be nice to have the same actions available when right-clicking the tab of the file as well.

**Comment by gldtn** [comment]
Guessing duplicate is not available yet? I don't see it when I right click a file. For now I just do the old copy/paste!
